src/main/webapp/lectures.js

```diff
@@ -1,5 +1,6 @@
 /* Used to gather URL parameters. */
 const URL_PARAM_ID = 'id';
+const URL_PARAM_VIDEO_ID = 'videoId';
```

Comment:
`'video-id'` to match other url/request params?

Reply:
fixed